events/dbpersist.go

```diff
@@ -34,26 +34,14 @@ type RepoEventRecord struct {
 	Event  string
 	Rebase bool
 
-	Ops []RepoOpRecord
-}
-
-type RepoOpRecord struct {
-	ID                uint `gorm:"primarykey"`
-	RepoEventRecordID uint `gorm:"index"`
-	Path              string
-	Action            string
-	Rec               *util.DbCID
+	Ops []byte
```

Comment:
I think that stream CBOR size limits (a few MByte?) would prevent the ops list from being so large it results in a DB error. But it might be worth a sanity check anyways, that the JSON encoded bytes aren't more than a MByte or something like that?

IIRC we exclude "huge" events at the block (data) level, but don't remember if we include all the ops in that case. Eg, if you do a huge delete of 10k records in a single repo request (to PDS), does the full ops list get emitted?

Reply:
Yeah, near the top of the persist method we truncate ops to be max 8192 long (can make that more aggressive too)